src/js/player.js

```diff
@@ -395,6 +395,12 @@ class Player extends Component {
     // Init debugEnabled_
     this.debugEnabled_ = false;
 
+    // Init state audioOnlyMode_
+    this.audioOnlyMode_ = false;
+
+    // Init state hiddenPlayerChildren_
+    this.hiddenPlayerChildren_ = [];
```

Comment:
Since this property specifically relates to audio-only mode, I'd suggest changing the name to something like `audioOnlyHiddenChildren_`.

However, given my comment about `height` below, maybe an `audioOnlyCache_` object that can store arbitrary values for reverting out of audio-only mode?